hoomd/md/EvaluatorPairMie.h

```diff
@@ -86,12 +90,16 @@ class EvaluatorPairMie
                 v["n"] = m3;
                 v["m"] = m4;
 
-                Scalar sigma = pow(m1 / m2, 1 / (m3 - m4));
-                Scalar epsilon = m1 / pow(sigma, m3) * (m3 - m4) / m3 * pow(m3 / m4, m4 / (m4 - m3));
-
+                Scalar sigma = fast::pow(m1 / m2, 1 / (m3 - m4));
+                Scalar epsilon = m1 / fast::pow(sigma, m3) * (m3 - m4) / m3 * fast::pow(m3 / m4, m4 / (m4 - m3));
+		
+                Scalar n = fast::exp2(Scalar(2.0) * mie_log2_n);
+                Scalar m = fast::exp2(Scalar(2.0) * mie_log2_m);
```

Comment:
```suggestion
                Scalar n = fast::exp2(Scalar(2.0) * mie_log2_n);
                Scalar m = fast::exp2(Scalar(2.0) * mie_log2_m);
```
n and m are stored directly above:
```
v["n"] = m3;
v["m"] = m4;
```